problem5/calculate.c

```diff
@@ -36,6 +36,12 @@ int total(int numbers[], int length) {
 
 
 bool areStringsEqual(const char* first, const char* second) {
+
+    int strcmp(const char* first, const char* second);
```

Comment:
So here, the first thing to remember is that `strcmp` is a _function_. The other thing to remember here is that when you're calling a function in C, you don't need the variable types; you can just use the variable names. So this function call should look something like this:

```c
int result = strcmp(first, second);
```